imap_processing/ultra/l1b/lookup_utils.py

```diff
@@ -0,0 +1,158 @@
+"""Contains tools for lookup tables for l1b."""
+
+import numpy as np
+import pandas as pd
+
+from imap_processing import imap_module_directory
+
+base_path = f"{imap_module_directory}/ultra/lookup_tables"
+
+
+def get_y_adjust(dy_lut: np.ndarray):
+    """
+    Adjust the front yf position based on the particle's trajectory.
+
+    Instead of using trigonometry, this function utilizes a 256-element lookup table
+    to find the Y adjustment. For more details, refer to pages 37-38 of the
+    IMAP-Ultra Flight Software Specification document (7523-9009_Rev_-.pdf).
+
+    Parameters
+    ----------
+    dy_lut : np.ndarray
+        Change in y direction used for the lookup table (mm).
+
+    Returns
+    -------
+    yadj : np.ndarray
+        Y adjustment (mm).
+    """
+    yadjust_path = f"{base_path}/yadjust.csv"
```

Comment:
Specifying the path this way is not platform agnostic. I suggest the following equivalent change for all of these functions that read csv files.
```suggestion
    yadjust_path = base_path / "yadjust.csv"
```